test/install_wkhtmltoimage.sh

```diff
@@ -7,7 +7,8 @@ sudo apt-get -qq update
 
 sudo apt-get install -y xfonts-75dpi
 wget http://download.gna.org/wkhtmltopdf/0.12/0.12.2/wkhtmltox-0.12.2_linux-trusty-amd64.deb
-sudo dpkg -i wkhtmltox-0.12.2_linux-trusty-amd64.deb
+sudo dpkg -i wkhtmltox-0.12.2_linux-trusty-amd64.deb || true
```

Comment:
It was missing a dependency, which the next line resolves automatically.  I figured that if that doesn't work, then the tests will still fail when it tries to run `wkhtmltoimage`.